docs/streams/developer-guide/config-streams.html

```diff
@@ -265,6 +263,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
             <td>10485760</td>
           </tr>
           <tr class="row-even"><td>cache.max.bytes.buffering (Deprecated. Use statestore.cache.max.bytes instead.)</td>
+          <tr class="row-even"><td>cache.max.bytes.buffering</td>
```

Comment:
Why do we insert this line here? Seems like a duplicate of the exiting line right above it?